styleguide/derek/view-templates/resource-center/resource-center.ejs

```diff
@@ -1,3 +1,93 @@
+              <option value="102">Azure</option>
+            </select>
+          </div>
+          <h6 class="rsFilter-fieldTitle">Resource Type</h6>
+          <div class="form-item form-type-check form-item-tid-3">
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Webinar" value="Webinar">
+                <label class="rsFilter-formCheckLabel">Webinar</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="PDF" value="PDF">
+                <label class="rsFilter-formCheckLabel">PDF</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Code" value="Code">
+                <label class="rsFilter-formCheckLabel">Code</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Video" value="Video">
+                <label class="rsFilter-formCheckLabel">Video</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Datasheet" value="Datasheet">
+                <label class="rsFilter-formCheckLabel">Datasheet</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Support" value="Support">
+                <label class="rsFilter-formCheckLabel">Support</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="eBook" value="eBook">
+                <label class="rsFilter-formCheckLabel">eBook</label>
+            </div>
+            <div class="rsFilter-formCheckItem">
+              <input class="rsFilter-formCheck form-checkbox" type="checkbox" name="Whitepaper" value="Whitepaper">
+                <label class="rsFilter-formCheckLabel">Whitepaper</label>
+            </div>
+          </div>
+        <button class="rsFilter-button">Show Results</button>
+      </form>
+    </div>
+    <div class="rsFilter-content">
+      <div class="resourcecenter-resourceRow">
+        <%- partial('../../pattern-components/additional-resources/additional-resourcesNoContainer.ejs') %>
+      </div>
+      <div class="resourcecenter-resourceRow">
+        <%- partial('../../pattern-components/additional-resources/additional-resourcesNoContainer.ejs') %>
+      </div>
+      <div class="resourcecenter-resourceRow">
+        <%- partial('../../pattern-components/additional-resources/additional-resourcesNoContainer.ejs') %>
+      </div>
+    </div>
+  </div>
+</div>
+<script src="js/view-templates/events/events.js" charset="utf-8"></script>
```

Comment:
Should `js/view-templates/events/events.js` get renamed to something a little bit more global for consistency since it's now being used across sections?